appeals/web/src/server/appeals/appeal-details/costs/costs.controller.js

```diff
@@ -93,28 +93,42 @@ export const getDocumentUpload = async (request, response) => {
 		session,
 		currentAppeal,
 		currentFolder,
-		params: { costsApplicant }
+		params: { costsCategory }
 	} = request;
 
 	if (!currentAppeal) {
 		return response.status(404).render('app/404');
 	}
 	if (!currentFolder) {
-		return response.status(404).render('app/500');
+		return response.status(404).render('app/404');
```

Comment:
Can this go into the one above as a "OR"?
(!currentAppeal || !currentFolder)

Reply:
done